### ISSUE TITLE
A monitor scale change was detected. 

We recommend restarting Spyder to ensure that it's properly displayed. If you don't want to do that, please be sure to activate the option

Enable auto high DPI scaling

in Preferences > General > Interface, in case Spyder is not displayed correctly.

Do you want to restart Spyder

### ISSUE THREAD

**Issue by ericberlow** [spyder-ide/spyder#11979]
### What steps will reproduce the problem?

I can't 100% reproduce it, but it appears to be happening when I have spyder displayed an external monitor. But if I restart spyder I keep getting the same area.  The spyder prefereces are correctly set. 

### Traceback
```python-traceback
  File "/Users/elb/opt/anaconda3/lib/python3.7/site-packages/spyder/app/mainwindow.py", line 1393, in handle_new_screen
    self.show_dpi_change_message)
TypeError: 'method' object is not connected
```

## Versions

* Spyder version: 4.1.1 
* Python version: 3.7.0
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Darwin 19.4.0

### Dependencies

```

# Mandatory:
applaunchservices >=0.1.7      :  0.2.1 (OK)
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.3.0 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.4.4 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.9 (OK)
qdarkstyle >=2.8               :  2.8 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.1 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.3 (OK)
sphinx >=0.6.6                 :  2.4.0 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.15 (OK)
matplotlib >=2.0.0             :  3.1.3 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  1.0.2 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5.1 (OK)

# Spyder plugins:
spyder_notebook                :  0.2.3 (OK)
```

**Comment by ccordoba12** [comment]
Duplicate of #11903

**Comment by ccordoba12** [comment]
For next time there's no need to paste the entire contents of the error message in the issue title. That title is a summary of the problem, not the entire problem.

**Comment by ericberlow** [comment]
thanks. sorry about that. this was my first time ever reporting an issue on GitHub :)

**Comment by ccordoba12** [comment]
No worries, I imagined that. That's why I wanted to let you know what you should do for next time.

I also forgot to mention that this problem will be fixed in our next release (4.1.2), to be released this weekend.
